src/lib/pickles/plonk_checks/scalars.ml

```diff
@@ -1020,6 +1040,1680 @@ module Tick : S = struct
                        "0x40000000000000000000000000000000224698FC094CF91B992D30ECFFFFFFFB"
                    )
                  * cell (var (Witness 13, Curr)) ) ) )
+      ; ( Index ChaCha0
```

Comment:
This is autogenerated from the rust side!! I strongly recommend skipping during review!